lib/person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(0..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] Based on the requirements, kindly make this method a private method. A private method can only be called within a class. Private methods are used as a way of protecting the behaviours (methods) defined in a class from being called by other objects not in the same class, or inherited from the same class. Kindly implement this change. 👍 😎